requests-and-responses/intro-to-dns-ip.md

```diff
@@ -10,6 +10,10 @@ About 1 hour
 ### Prerequisites
 - [Command Line Lesson](/command-line/command-line-interface.md)
 
+### Motivation
+  
+Understanding about DNS is an important part of any website. Without it, one would have to remember the long IP addresses. But the concept of DNS has made it easier.
```

Comment:
@yati1998, you can present this line better:
Like,
Understanding about DNS is important when it comes to IP address or URL of a website. DNS plays an important role in communicating with a website, like, when we enter "www.google.com", do you know what really happens behind the scenes? Let's find out.

Yati, try to make the Motivation part as attractive as you can.

Reply:
okay sure.